Packetery/Checkout/view/adminhtml/web/css/source/_admin-menu.less

```diff
@@ -0,0 +1,18 @@
+@font-face {
+    font-family:'Packetery';
+    src:url('../fonts/packeta-logo.eot');
+    src:url('../fonts/packeta-logo.eot?#iefix') format('embedded-opentype'), url('../fonts/packeta-logo.woff') format('woff'), url('../fonts/packeta-logo.ttf') format('truetype'), url('../fonts/packeta-logo.svg') format('svg');
```

Comment:
toto bych lépe naformátoval.  jaký je důvod mít to logo ve všech těchto formátech ?